client/post-editor/editor-publish-date/post-scheduler.jsx

```diff
@@ -9,35 +9,28 @@ import { get } from 'lodash';
 /**
  * Internal dependencies
  */
-import AsyncLoad from 'components/async-load';
+import PostSchedule from 'components/post-schedule';
 import QueryPosts from 'components/data/query-posts';
 import postUtils from 'lib/posts/utils';
 import siteUtils from 'lib/site/utils';
 import { getSitePostsForQueryIgnoringPage } from 'state/posts/selectors';
 
-const PostSchedule = ( { onDateChange, onMonthChange, posts, selectedDay, site } ) => {
-	const tz = siteUtils.timezone( site );
-	const gmtOffset = siteUtils.gmtOffset( site );
-
+const PostScheduleWithCurrentMonthPosts = connect(
+	( state, { site, query } ) => ( {
+		posts: getSitePostsForQueryIgnoringPage( state, get( site, 'ID' ), query ) || [],
+	} )
+)( function( { onDateChange, onMonthChange, posts, selectedDay, site } ) {
 	return (
-		<AsyncLoad
```

Comment:
Why is AsyncLoad no longer needed? (Not sure why is was in there to begin with)

Reply:
Now that the publish confirmation sidebar is live, we're not using it in current `master`.

Investigating further, it was introduced here:  https://github.com/Automattic/wp-calypso/pull/9112

and the code size for that chunk is larger than I would have expected.

I don't have a clear preference for whether or not to bring back this behavior.  Thoughts?

Reply:
Okay, it's probably fine to leave it out, since we we not using `AsyncLoad` anymore in this context.

Reply:
:+1: I created https://github.com/Automattic/wp-calypso/issues/17924 for future follow-up there.